src/components/widgets/status/StatusTab.vue

```diff
@@ -18,16 +18,35 @@
           v-if="progressVisible && $vuetify.breakpoint.lgAndUp"
           cols="auto"
           align-self="center"
+          class="position-relative"
         >
-          <v-progress-circular
-            :rotate="-90"
-            :size="90"
-            :width="7"
-            :value="estimates.progress"
-            color="primary"
-          >
-            <span class="percentComplete focus--text">{{ estimates.progress }}%</span>
-          </v-progress-circular>
+          <div class="progress-wrapper">
+            <v-progress-circular
+              :rotate="-90"
+              :size="90"
+              :width="7"
+              :value="estimates.progress"
+              color="primary"
+              class="progress-circle"
+              sytle="pointer-events: none;"
+            />
+            <div class="thumbnail-background" />
+
+            <v-btn
+              text
+              height="100%"
+              class="print-thumb"
+              @click="handleViewThumbnail"
+            >
+              <img
+                class="print-thumb"
+                :src="thumbnail"
+              >
+            </v-btn>
+          </div>
```

Comment:
Enlarging the thumbnail doesn't work for me, this might need to be something like:

```suggestion
          <div
            class="progress-wrapper"
            style="cursor: pointer"
            @click="handleViewThumbnail"
          >
            <v-progress-circular
              :rotate="-90"
              :size="90"
              :width="7"
              :value="estimates.progress"
              color="primary"
              class="progress-circle"
            />
            <div class="thumbnail-background" />

            <div class="fill-height print-thumb">
              <img
                class="print-thumb"
                :src="thumbnail"
              >
            </div>
```

Reply:
@matmen  you can commit this to branch and I can test it out.